sdk/client.ts

```diff
@@ -1,18 +1,18 @@
 // import fetch from 'node-fetch';
 import {
   ClientBuilder,
-  
+
   type Client,
   type AuthMiddlewareOptions,
   type HttpMiddlewareOptions
- } from '@commercetools/sdk-client-v2'
+ } from '@commercetools/ts-client'
 
  const date = new Date().toISOString()
 
 // Configure httpMiddlewareOptions
 const httpMiddlewareOptions: HttpMiddlewareOptions = {
   host: 'https://api.europe-west1.gcp.commercetools.com',
-  fetch,
+  httpClient: fetch,
```

Comment:
Does this compile? The import for `fetch` is commented out

Reply:
Please clean up these docs to be more clear whether `node-fetch` is the preferred httpclient, see https://docs.commercetools.com/sdk/ts-sdk-getting-started

Also remove commented out import at top of this file for `node-fetch` if it is not required

Reply:
Yes it does compile, however you will need nodejs version 18, where we have native fetch built in. For the documentation, I will go ahead and update the docs.

Thanks